countingvowels1.py

```diff
@@ -1,4 +1,5 @@
-nname = input("Today you'll get to learn how many vowels are in the a sentence/word you input. Type your word/sentence here: ")
+name = input("Today you'll get to learn how many vowels are in the a sentence/word you input. Type your word/sentence here: ")
```

Comment:
Good catch with the misnamed variable - but this still mentions that it only counts vowels! It should mention that it counts vowels and consonants.